servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/BaseCommand.java

```diff
@@ -29,8 +30,29 @@ public abstract class BaseCommand implements Callable<Integer> {
   @Inject VersionStoreConfig versionStoreConfig;
   @Inject ServerConfig serverConfig;
 
+  @Inject Instance<DatabaseAdapter> databaseAdapterInstance;
```

Comment:
Should `@Inject` on line 29 be removed?

Reply:
Nope, that's the "quarkus specific" one

Reply:
... but we overwrite it from `databaseAdapterInstance.get()` in `call()` :thinking: 

What is the difference between these two adapter objects?

Reply:
err, wrong line 29